modules/launcher/src/main/java/org/mule/module/launcher/domain/DomainMuleContextBuilder.java

```diff
@@ -41,7 +41,7 @@ public DomainMuleContextBuilder(String domainId)
     @Override
     protected MuleConfiguration getMuleConfiguration()
     {
-        DefaultMuleConfiguration defaultMuleConfiguration = new DefaultMuleConfiguration();
+        DefaultMuleConfiguration defaultMuleConfiguration = new DefaultMuleConfiguration(true);
```

Comment:
if I migrate to a version containing this fix, does it mean that a new .mule will be created on MULE_HOME and whatever data stored in CWD will no longer be accessible? If so, shouldn't we include a migration guide entry saying that and instructing how to move the data over?